passes/techmap/abc.cc

```diff
@@ -1471,7 +1471,12 @@ struct AbcPass : public Pass {
 		po_map.clear();
 
 #ifdef ABCEXTERNAL
-		std::string exe_file = ABCEXTERNAL;
+		std::string exe_file;
+		if (std::getenv("ABCEXTERNAL")) {
```

Comment:
This should definitely not be called `ABCEXTERNAL`. The convention would be to use `ABC`, like you'd have `CC`, `CXX`, and so on.

Reply:
Done.